src/MetaModels/DcGeneral/Data/Driver.php

```diff
@@ -569,13 +569,15 @@ public function resetFallback($strField)
      * If the item does not have an Id yet, the save operation will add it as a new row to the database and
      * populate the Id of the model accordingly.
      *
-     * @param ModelInterface $objItem The model to save back.
+     * @param ModelInterface $objItem   The model to save back.
+     *
+     * @param int            $timestamp Optional the timestamp.
      *
      * @return ModelInterface The passed model.
      *
      * @throws \RuntimeException When an incompatible item was passed, an Exception is being thrown.
      */
-    public function save(ModelInterface $objItem)
+    public function save(ModelInterface $objItem, $timestamp = 0)
```

Comment:
The tstamp should be passed to `$objItem->getItem()->save();` below!

Reply:
We also should trigger a
```php
@trigger_error('Not passing a timestamp has been deprecated and will cause an error in MetaModels 3', E_USER_DEPRECATED);
```

Reply:
I have taken note of this